cspell.js

```diff
@@ -0,0 +1,64 @@
+exports.cspell = {
+    "words": [
+        "merkle",
+        "merklize",
+        "merklized",
+        "merklizer",
+        "merkletree",
+        "merklization",
+        "iden",
+        "creds",
+        "cred",
+        "claims",
+        "ciphertext",
+        "ldcontext",
+        "ZKPexperiance",
+        "polygonid",
+        "ipfs",
+        "privado",
+        "zeroknowledge",
+        "testdata",
+        "thid",
+        "groth16",
+        "reqs",
+        "secp",
+        "Eddsa",
+        "mtpv2",
+        "sigv2",
+        "authv",
+        "uuidv",
+        "sigv",
+        "mtpv",
+        "esrs",
+        "transak",
+        "multibase",
+        "onchain",
+        "credential",
+        "keccak",
+        "hindex",
+        "hvalue",
+        "metas",
+        "methodid",
+        "circom",
+        "XSDNS",
+        "nonbetween",
+        "crosschain",
+        "metadatas",
+        "wtns"
+    ],
+    "ignoreRegExpList": [
+      "Base64",
+      "Base64SingleLine",
+      "Base64MultiLine",
+      "HexValues",
+      "HashStrings",
+      "Urls",
+      "/seed\w*/g"
+    ],
+    "ignoreWords": [
+      "unmarshal",
+      "JUvpllMEYUZ2joO59UNui_XYDqxVqiFLLAJ8klWuPBw",
+      "gdwj",
```

Comment:
I guess better to remove `JUvpllMEYUZ2joO59UNui_XYDqxVqiFLLAJ8klWuPBw`, `gdwj`, `fwor` from this config and add ignore in the file explicitly

Reply:
removed